src/registrar/templates/application_dotgov_domain.html

```diff
@@ -48,16 +48,15 @@ <h2>What .gov domain do you want?</h2>
       {% endwith %}
     {% endwith %}
     <button
-      id="check-availability-button"
       type="button"
-      class="usa-button"
+      class="usa-button usa-button--outline"
       validate-for="{{ forms.0.requested_domain.auto_id }}"
     >Check availability</button>
   </fieldset>
 
   {{ forms.1.management_form }}
 
-  <fieldset class="usa-fieldset margin-top-1">
+  <fieldset class="usa-fieldset margin-top-1 dotgov-domain-form" id="form-container">
```

Comment:
(nitpick/optional) Not really sure if its worth changing, but "form-container" seems a little generic for the id. Ideally I like to be as specific as reasonably possible with ids because it both indirectly gives information about the object you are targeting, and it also reduces the chance of accidental id conflicts (where two elements on the page share the same id).

While its more verbose, I would recommend an id like `form-container-alternative-domain`. That said though this is totally optional to implement and more of a stylistic thing, so feel free to pass over this one

Reply:
The enhanced forms all use that id. Changing it will require a small JS refactor, I'd say leave it alone for now